_data/simple-icons.json

```diff
@@ -385,6 +385,11 @@
             "hex": "000000",
             "source": "http://codepen.io"
         },
+        {
+          "title": "CodeSandbox",
+          "hex": "000000",
```

Comment:
Can you provide a source or reason for choosing this color?

Reply:
In that regard, if I look at their social media ([Twitter](https://twitter.com/codesandbox) and [Spectrum](https://spectrum.chat/codesandbox)), I would expect the color to be white 🤔

Reply:
Am I misunderstanding our format? Is the color in the JSON the fill color of the icon brand-wise? Or the background color of the icon displayed on [SimpleIcons](https://simpleicons.org/)?

Reply:
The color *should* be the "main" brand color and it is displayed on the website as the background color (although that is subject to change, see #980). The fill color of the SVG on the website is always black or white.  When you download the icon from the website it has no color specified.